### ISSUE TITLE
Specify Job Regex

### ISSUE THREAD

**Issue by dcendents** [zionyx/jenkins-tray#69]
Hi,

Great improvements, I've been using hudson tray for a while and I'm happy to see you are now maintaining a Jenkins Tray project.

One feature I always missed was the ability to specify a regex to select which jobs I'm interested to follow. A similar feature to views in Jenkins which allow you to select jobs with a simple regex.

Every time the application refresh the jobs status, I'd expect new jobs that match the regex to be automatically selected.

Let me know if this is something you could add.

Thanks

**Comment by dcendents** [comment]
I would have loved to help, I got VisualStudio installed at home and was ready to give it a go but then I got the DevExpress window poping and every time I try to open the settings UI page VisualStudio crash.

I hope you find the time to add this feature, it would really help our team.

**Comment by zionyx** [comment]
Thanks for the kind words.

However, i think you need to install the right DevExpress version to
develop on it. Not too sure if you can use an evaluation license for that,
most likely possible.

It is a real shame that DevExpress is counter productive in that way. If
any workaround on the UI side can help you get going, please let me know.
Thanks.

BR
Lee

**Comment by dcendents** [comment]
Yes please, if you could create a branch and modify the UI to add a regex field I could probably do the rest. 

What I had in mind was something quite simple like on the jenkins view configuration.

![viewregex](https://cloud.githubusercontent.com/assets/595214/11616454/7b7bddd6-9c49-11e5-9970-b0c29061044e.png)

I think the Settings dialog where we select the jobs is the right place to put it. The way it works in Jenkins, you can still select individual jobs and also specify a regex to add more jobs, so I was thinking of doing the same behavior.
Let me know if that works for you.

Cheers,
Daniel

**Comment by zionyx** [comment]
Sorry for the late reply. Just got back from an escape with family for a week.
Hmm. I'll try to copy as much standard view options as possible to the UI and let you know after the weekend. Or if you have a more specific UI in mind, let me know in detail. Either way works. Thanks!

**Comment by zionyx** [comment]
I'll find some time to make this UI and branch for you within this month. It's been a long break and been busy with work since then. Thanks!

**Comment by dcendents** [comment]
Hi,

sorry it's been the same for me so I understand very well. 
If you can modify the UI I can definitely do the rest.

Here is a modified screenshot on how I see it, but feel free to do it differently if you prefer:

![serversettings](https://cloud.githubusercontent.com/assets/595214/12283156/27c17f2a-b972-11e5-86b5-249861c05cea.png)

Cheers,

**Comment by zionyx** [comment]
Thanks. I'll add some additional options to mimic what Jenkins CI offers. Will let you know when I have done it. May take a while. Cheers.

**Comment by zionyx** [comment]
https://github.com/zionyx/jenkins-tray/issues/78
